app/models/resource_action.rb

```diff
@@ -19,6 +19,12 @@ def automate_queue_hash(target, override_attrs, user)
       override_values = {}
     elsif target.kind_of?(Hash)
       override_values = target
+    elsif target.kind_of?(Array) || target.kind_of?(ActiveRecord::Relation)
+      klass = target.first.id.class
+      object_ids = target.collect { |t| "#{klass}::#{t.id}" }.join(",")
+      override_attrs = {:array_object_type  => target.first.class.base_class.name,
+                        'Array::object_ids' => object_ids}.merge(override_attrs || {})
```

Comment:
@gmcculloug the override_attrs can also be nil so we have to initialize it before we do the merge